### ISSUE TITLE
Imports run as background jobs

### ISSUE THREAD

**Issue by mhulet** [extinctionrebellion/RebelsManager#65]

**Comment by begault** [comment]
Hello @mhulet 
Do you want to use a sidekiq process or something similar to treat this issue ? 
It can be done through a worker creating jobs in a sidekiq queue. 
Is it what you're waiting for ? 
If so, I can work on it.

**Comment by mhulet** [comment]
Yes @begault, nothing fancy. I'd go for Sidekiq, that we'll quickly use for calls to the Mailtrain API too.

**Comment by begault** [comment]
Oops sorry, I did not see this issue was already assigned. 
Great ! So, I'll focus on data imports.
